shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -30,6 +44,14 @@ const AddMembersConfirm = () => {
       addingMembers.findIndex(member => !member.assertion.endsWith('@email')) === -1,
     [addingMembers]
   )
+  const emailsAndPhones = React.useMemo(
```

Comment:
We should check for any non keybase user, twitter assertions and such

Reply:
I forgot you could get other assertions here